pkg/commands/logging/newrelic/newrelic_test.go

```diff
@@ -64,7 +64,7 @@ func TestNewRelicCreate(t *testing.T) {
 					}, nil
 				},
 			},
-			Args:       args("logging newrelic create --key abc --name foo --service-id 123 --version 3"),
+			Args:       args("logging newrelic create --key abc --name foo --service-id 123 --version 3 --region eu"),
 			WantOutput: "Created New Relic logging endpoint 'foo' (service: 123, version: 3)",
```

Comment:
I would prefer to remove this flag from the test as we're not able to validate anything other than the flag has been defined (this is why I omitted the other flags and only included flags whose content we could see in the response object and would be displayed back to the user in the success message).